src/main/java/com/pnu/dev/radioserviceapi/client/dto/YoutubeApiResult.java

```diff
@@ -0,0 +1,32 @@
+package com.pnu.dev.radioserviceapi.client.dto;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Data;
+import org.apache.commons.lang3.StringUtils;
+
+@Data
+@Builder(access = AccessLevel.PRIVATE)
+public class YoutubeApiResult<T> {
+
+    T data;
```

Comment:
Both fields should be private, as well as class should be named OperationResult and located under `util` package (because, it can be reused with another flows)